eng/testing/tests.singlefile.targets

```diff
@@ -142,19 +142,6 @@
     </PropertyGroup>
   </Target>
 
-  <Target Name="__ReplaceCrossgen2ExecutableWithFreshlyBuiltOne"
-          BeforeTargets="_PrepareForReadyToRunCompilation">
-    <PropertyGroup>
-      <Crossgen2ArtifactPath>$(CoreCLRCrossgen2Dir)crossgen2$(ExeSuffix)</Crossgen2ArtifactPath>
-    </PropertyGroup>
-
-    <ItemGroup>
-      <Crossgen2CurrentTool Include="@(Crossgen2Tool->'$(Crossgen2ArtifactPath)')" />
-      <Crossgen2Tool Remove="@(Crossgen2Tool)" />
-      <Crossgen2Tool Include="@(Crossgen2CurrentTool)" />
-    </ItemGroup>
-  </Target>
-
```

Comment:
@ivdiazsa I've removed this target now that we automatically hook up the Crossgen2 tool when referencing the live build.